ppx/ppx_deriving_rpc.cppo.ml

```diff
@@ -73,6 +73,8 @@ let attr_name  = attr_string "name"
 (* Documentation for variants / record members *)
 let attr_doc = attr_string "doc"
 
+let is_dict attr = match Ppx_deriving.attr ~deriver "dict" attr with Some _ -> [%expr true] | None -> [%expr false]
```

Comment:
You want to make my life harder when I drop the `ppx_deriving` dependecy 😆

Reply:
Oh dear, really?

Reply:
No worries. I still hope we won’t need to drop it